include/fastdds/dds/core/Entity.hpp

```diff
@@ -124,10 +124,10 @@ class Entity
      * @brief Allows access to the StatusCondition associated with the Entity
      * @return Reference to StatusCondition object
      */
-    RTPS_DllAPI const StatusCondition& get_statuscondition() const
+    RTPS_DllAPI StatusCondition& get_statuscondition() const
     {
         logWarning(CONDITION, "get_statuscondition method not implemented");
-        return status_condition_;
```

Comment:
Why removing the const in the return?

Reply:
To allow doing `entity.get_statuscondition().set_enabled_statuses(enabled_statuses)`